hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/troubleshooting_s3a.md

```diff
@@ -1179,7 +1179,8 @@ file using configured SSE-C keyB into that structure.
 ### Instruction file not found for S3 object
 
 Reading an unencrypted file would fail when read through CSE enabled client.
-```
+
+```txt
```

Comment:
Why?

Reply:
because typically code blocks in markdown indicate the type of content in it.
and i thought while i was at it i could change this line.

Reply:
Good intentions, but we practice keeping things isolated.

in case one thing has to be reverted or backported it can be done without bothering the other

Reply:
sure i am reverting it.